daprdocs/content/en/developing-applications/building-blocks/workflow/howto-manage-workflow.md

```diff
@@ -58,6 +70,41 @@ To terminate your workflow with an ID `12345678`, run:
 POST http://localhost:3500/v1.0-alpha1/workflows/dapr/12345678/terminate
 ```
 
+### Raise an event
+
+Run the following for your workflow to create "wait for external event" tasks that:
+
+1. Subscribe to external events.
+1. Await those tasks to block execution until the event is received.
+
+```bash
+POST http://localhost:3500/v1.0-alpha1/workflows/<workflowComponentName>/<instanceID>/raiseEvent/<eventName>
+```
+
+> An `eventName` can be any function. 
+
+### Pause or resume a workflow
+
+To plan for down-time, wait for inputs, and more, you can pause and then resume a workflow. To stop a workflow with an ID `12345678` until triggered to resume, run:
+
+```bash
+POST http://localhost:3500/v1.0-alpha1/workflows/dapr/12345678/pause
+```
+
+To resume a workflow with an ID `12345678`, run:
+
+```bash
+POST http://localhost:3500/v1.0-alpha1/workflows/dapr/12345678/resume
+```
+
+### Purge a workflow 
+
+To remove actor state from your state store and avoid overloading your state stores, run:
+
+```bash
+POST http://localhost:3500/v1.0-alpha1/workflows/dapr/OrderProcessingWorkflow/purge
```

Comment:
"OrderProcessingWorkflow" should be "12345678" since we purge based on the instance ID, not the workflow name.